### PR TITLE
Signup signin

### PR DESCRIPTION
**Pull request by Deji98**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: dma_frontend/src/components/SignInComp.jsx [hunk 1]
```diff
@@ -1,11 +1,69 @@
...
-
+import '../styles/signin.css';
+import signin from '../images/signin.jpg'
 const SignInComp = () => {
   return (
     <div className=''>
       <div className=''>
-        {/* TYPE YOUR CODE HERE. IF YOU USE TAILWINDCSS MAKE SURE TO ADD THE tw- PREFIX TO THE UTILITY CLASS AND IF YOU WRITE VANILLA CSS KINDLY USE login.css FILE */}
+        {         <div class="in-container">
+         <div class="in-item-2">
```
Comment (hokagedemehin) [inline dma_frontend/src/components/SignInComp.jsx:9]: Please check through this whole file and change any html entity that has `class` to `className`.  

I only highlighted this section as an example for you. please look through the whole file and change anywhere this kind of issue appears

File: dma_frontend/src/components/SignInComp.jsx [hunk 2]
```diff
@@ -1,11 +1,69 @@
...
+               </a> */}
+             </div>
+           </div>
+           <img id="img-desktop-in" src="/svg/svg" alt="" />
+           <p id="p-in">Sign In</p>
+ 
+           <form class="forme">
+ 
+             <label for="email">Email address*</label><br /><br />
+             <input
```
Comment (hokagedemehin) [inline dma_frontend/src/components/SignInComp.jsx:32]: Please check through this whole file and change any label entity that has `for` to `htmlFor`.  

I only highlighted this section as an example for you. please look through the whole file and change anywhere this kind of issue appears

File: dma_frontend/src/components/SignInComp.jsx [hunk 3]
```diff
@@ -1,11 +1,69 @@
...
+             /><br /><br /><br />
+ 
+             <label for="pswd">Password*</label><br /><br />
+             <input
+               type="password"
+               name="password"
+               placeholder="******************"
+               class="pswd01-in"
+               autocomplete="current-password"
+               required=""
```
Comment (hokagedemehin) [inline dma_frontend/src/components/SignInComp.jsx:47]: Change `autocomplete` to `autoComplete`

**Review by hokagedemehin — CHANGES_REQUESTED** [review]

**Comment by Deji98** [comment]
Okay.
Will work on that in the evening
Thanks

**Pull request merged**